### DISCUSSION TITLE
部分竖屏视频播放黑屏

### DISCUSSION THREAD

**Discussion by Lornight** [ywmoyue/biliuwp-lite#450] [General]
### Bug 描述

部分竖屏视频播放黑屏

### 操作步骤

```plain text
点进部分竖屏的视频
```

### 期望结果

正常播放

### 截图

![}K{2B2Q6M{I3B0DQFZQ`O00](https://github.com/ywmoyue/biliuwp-lite/assets/73097211/377d1b06-300b-463e-bac7-67fd6fe06950)

BV1WC4y1D7uh

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

Windows 11 21H2 (Build 22000), 其它 (需注明)

### 其它系统版本

win11 22h2

### 设备类型

台式机

### 设备架构

x64

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
请给出这个视频的地址或者给出日志，这有助于bug修复，感谢。

**Comment by Lornight** [comment]
https://b23.tv/BV1WC4y1D7uh
这个是视频链接




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月22日(星期五) 晚上8:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ywmoyue/biliuwp-lite] 部分竖屏视频播放黑屏 (Discussion #450)





 
请给出这个视频的地址或者给出日志，这有助于bug修复，感谢。

**Comment by GD-Slime** [comment]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/82302542/1fd2af2f-5c6a-48d0-9041-bf1a250fe0da)
我这里是可以正常播放的。能否点击一下右边三个点看一下信息呢？我怀疑是你可能在使用一些没有安装相应扩展的视频编码。

**Reply by Lornight** [reply]
av1的扩展我安装了，试了一下就只有h264能看不知道为什么
![S3 7FC}5605R3KWB48W`0ST](https://github.com/ywmoyue/biliuwp-lite/assets/73097211/5cea36a6-28ab-42a6-aa47-fe1af3cd10ec)

**Reply by GD-Slime** [reply]
不是所有视频都有av1的, 当没有av1时就会回落到h265. 图中这个视频正好只有h264和h265. 可以在设置 -> 播放中找到安装h265(hevc)的办法.

**Reply by Lornight** [reply]
问题已解决，非常感谢
